src/export/StructureDefinitionExporter.ts

```diff
@@ -493,6 +502,64 @@ export class StructureDefinitionExporter implements Fishable {
     return this.fisher.fishForMetadata(item, ...types);
   }
 
+  applyElementProfiles(caretRules: CaretValueRule[], structDef: StructureDefinition): void {
+    const profileElementExtension =
+      'http://hl7.org/fhir/StructureDefinition/elementdefinition-profile-element';
```

Comment:
According to those rule followers (cough @cmoesel cough) we should make this a constant at the top of the file I think? Alternatively I think in this case you could just use that string directly where it is used, since it only appears to be used one place.